### ISSUE TITLE
ct3_rt: Can't set long node name!

### ISSUE THREAD

**Issue by matwey** [rebar/rebar#474]
Hi,

Do we really need following code on line 43 in ct3_rt.erl?
```
 41 run(_Dir) ->
 42     {ok, _} = retest:sh("./rebar compile ct -v",
 43                         [{env, [{"ERL_FLAGS", "-name ct_rt3"}]}]),
 44     ok.
```

On badly configured hosts (without proper FQDN), this leads to test failure. Maybe this can be dropped or changed to -sname?

**Comment by tuncer** [comment]
Not strictly required, feel free to submit a patch.

**Comment by lrascao** [comment]
This has actually been fixed indirectly by way of #562, the approach chosen (that also works on Windows) was setting the full node name (suffixed with @localhost), https://github.com/rebar/rebar/pull/562/files#diff-bfab0bebbc294af8b0b15e47cfa4df5dL43
